### ISSUE TITLE
在JupyterLab中,print(c)和c的输出一致,他们有什么区别?

### ISSUE THREAD

**Issue by feifanruwo** [neolee/pilot#1355]
![image](https://user-images.githubusercontent.com/66456065/86530162-004e2480-bee9-11ea-976d-1df0c61d77c4.png)

**Comment by neolee** [comment]
对 Python 解释器来说，`print(c)` 的意思是“在标准输出界面打印 c 的值”，而单独一个 `c` 意思是“计算 c 的值”——并没有输出和显示的意思。

在 Jupyter Lab 和 Python 解释器中单独运行一个 `c` 会显示出 c 的值，这是 Jupyter Lab 和解释器界面专门实现的一个功能，是为了方便编程者随时查看某个变量的值，你可以理解为工具性质的，而不是语言本身具有的特性。

如果上面这些还不好理解，也没关系，总之不太确定的情况下，还是用 `print()` 来输出显示吧。

**Comment by feifanruwo** [comment]
3Q
(请问我提出问题得到老师解答之后给出感谢,这么做会有打扰之嫌吗,是不是有些多余?)

**Comment by neolee** [comment]
打扰谈不上，但确实不必要。

问题解决之后关掉问题就足够了。
